src/services/completions.ts

```diff
@@ -91,6 +91,7 @@ import {
     getNameOfDeclaration,
     getNameTable,
     getNewLineCharacter,
+    getNewLineOrDefaultFromHost,
     getNewLineKind,

```

Comment:
This lint is funny because the spirit of this lint is keep things "roughly in the same place" but I wouldn't hold a linter to it 8 letters in.

Reply:
The import formatting PR at least enables organize to work now (plus or minus a bug I'm looking at).